applications/GeoMechanicsApplication/cpp_geomechanics_application.cpp

```diff
@@ -0,0 +1,918 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Jonathan Nuttall
+//
+
+#pragma once
+
+#include "cpp_geomechanics_application.h"
+#include <processes/apply_constant_scalarvalue_process.h>
+#include <utilities/read_materials_utility.h>
```

Comment:
We prefer `<>` includes to only be used for standard/system libs. Is there any reason to have them as `""`?

*Edit*: Since I saw this in more files, in general we prefer `<>`only for std includes, and if possible always before project includes, since they are often slow to parse.